CHANGES.rst

```diff
@@ -53,6 +53,12 @@ Features added
   GMT (universal time) instead of local time for the date-time
   supplied to :confval:`html_last_updated_fmt`.
   Patch by Adam Turner.
+* #12910: Copyright entries now support the ``'%Y'`` placeholder
+  to substitute the current year.
+  This is helpful for reducing the reliance on Python modules
+  such as :py:mod:`time` or :py:mod:`datetime` in :file:`conf.py`.
+  See :confval:`the docs <copyright>` for further detail.
```

Comment:
Typo/broken reference here?

Reply:
All seems fine:

https://sphinx--12910.org.readthedocs.build/en/12910/changes/index.html#features-added

However it is ugly & this prompted me to check